docs/docs-content/clusters/edge/local-ui/cluster-management/share-auth.md

```diff
@@ -0,0 +1,155 @@
+   depending on your application.
+
+   For example, the following is an implementation of an [Express.js](https://expressjs.com/) middleware that can be
+   used to verify the JWT before your server responds to an incoming HTTP request.
+
+   ```javascript
+   function checkAuth(req, res, next) {
+     const token = req.cookies["Authorization"];
+     if (!token) {
+       return res.redirect(getAuthURL(req)); // Redirect to the login screen if no token is detected
+     }
+
+     jwt.verify(token, secretKey, (err, decoded) => {
+       if (err) {
+         console.log("Error verifying token", err);
+         return res.redirect(getAuthURL(req)); // Redirect to the login screen if the token is invalid
+       }
+       req.user = decoded; // Return the decoded user information if the token is valid
+       next();
+     });
+   }
+   ```
+
+   The middleware verifies the JWT using the public key. If it is valid, it decodes the JWT and return the user
+   information contained in the JWT. If the JWT doesn't exist or is not valid, it will redirect the user to the login
+   page.
+
+3. Build the image containing your application and push it to a registry that your host has access to.
+
+4. Create a Kubernetes manifest that you can use to deploy your containerized application.
+
+5. In the manifest for your application, define a volume that contains a Kubernetes secret. The secret name is
+   `<edge-host-ID>-public-key`. Replace `<edge-host-ID>` with the ID of your Edge host. This secret contains the public
+   key that your application will use to verify the token.
+
+   :::info
+
+   To find your Edge host ID, you can log in to [Local UI](../host-management/access-console.md). In the **Overview**
+   section is the Edge host ID. If you didn't enable Local UI, you can log in to
+   [Palette](https://console.spectrocloud.com) and find the Edge host ID in the
+   [Edge host grid view](../../site-deployment/edge-host-view.md) page in the **Machine ID** column.
+
+   :::
+
+6. In the manifest for your application, mount the volume containing the secret to the location specified in your
+   application. For example, if your application is trying to access `/app/keys/server.pem`, you should mount the volume
+   to `/app/keys/`.
+
+   If you have previously used an environment variable to store the path of the public key, define the corresponding
+   environment variable in your container specification. For example, if your application will try to access an
+   environment variable `KEY_LOCATION`, you can define the environment variable using configurations similar to the
+   following.
+
+   ```yaml
+   spec:
+     containers:
+       - name: sample-auth-app
+         image: gcr.io/example-app:latest
+         volumeMounts:
+           - name: jwt-secret
+             mountPath: /app/keys
+             readOnly: true
+         env:
+           - name: KEY_LOCATION
+             value: "/app/keys/server.pem"
```

Comment:
@lennessyy If we are gonna use the volumeMount example, then we should also add the container volume definition. Otherwise the example is missing the key point of reference, the actually secret 🙈 

```
      volumes:
      - name: jwt-secret
        secret:
          # change this to match the edge name
          secretName: edge-2ec13642c85aba0f5306230c9c25156f-public-key
```